server/gc/src/main/java/org/apache/accumulo/gc/GCRun.java

```diff
@@ -318,7 +321,7 @@ public void deleteConfirmedCandidates(SortedMap<String,String> confirmedDeletes)
             processedDeletes.add(delete);
           }
         } catch (Exception e) {
-          log.error("{}", e.getMessage(), e);
+          log.error("{} {}", fileActionPrefix, e.getMessage(), e);
```

Comment:
Usually when we put in the `e.getMessage()`, it's redundant if we put in the stack trace. So, since the stack trace is included here, we can just do something like:

```suggestion
          log.error("{} Exception while deleting files ", fileActionPrefix, e);
```